Lesson21/todo-app/sahra/styles.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+:root{
+  --rose-quartz:#F2BCBA;
+  --charcoal-black:#202221;
+  --beige-mist:#EFE1C7;
+  --checked-icon:#efe1c783;
+  --remove-icon:#2c2d2c;
+}
+body {
+  margin: 0;
+  background: linear-gradient(90deg, var(--rose-quartz), var(--beige-mist));
+  min-height: 100vh;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  margin: 50px auto;
+  padding: 20px;
+  background-color: var(--charcoal-black);
+  border-radius: 5px;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
+  max-width: 600px;
+}
+
+h1 {
+  text-align: center;
+  color: var(--beige-mist);
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+input[type="text"]{
+  flex: 1;
+  padding: 0px 10px 0px 10px;
+  font-size: 16px;
+
+}
+.todo-form-item{
```

Comment:
Combine input[type="text"] and .todo-form-item selectors.
Make the input element a bit bigger by adding padding. Ideally, it would be better to have input and submit button in a similar size.